packages/mobile/src/screens/notifications-screen/Notification/ProfilePictureList.tsx

```diff
@@ -26,20 +56,49 @@ type ProfilePictureListProps = {
 }
 
 export const ProfilePictureList = (props: ProfilePictureListProps) => {
-  const { users, style, navigationType, interactive, imageStyles } = props
-  const styles = useStyles()
+  const {
+    users,
+    totalUserCount = users.length,
+    limit = USER_LENGTH_LIMIT,
+    style,
+    navigationType,
+    interactive,
+    imageStyles
+  } = props
+  const styles = useStyles({
```

Comment:
def a small optimization, which is that every re-render will cause useStyles to get called, we can prevent that by useMemo'ing the useStyles params, not the cleanest, but best we have for now...

Reply:
cool thx